docker-compose.yml

```diff
@@ -123,6 +123,17 @@ services:
     env_file:
       - ./analytics/env.docker
 
+  proxy:
```

Comment:
I wonder if adding a proxy locally is the simplest approach. Could we alternatively add the certificates to the built Django container and use [gunicorn's SSL support](https://docs.gunicorn.org/en/stable/settings.html#ssl)?

I suppose that doesn't approximate the production environment and wouldn't solve the issue in production. In that sense I am curious what @rbadillap will say, whether it's worth using SSL even with a proxy (I don't see how it could hurt, and if anything seems wise considering we don't _actually_ control the infrastructure running in AWS and have no idea what HTTP routing in AWS's internal networks is actually doing; that might be overly paranoid though).

Reply:
Yeah, the main reason for using NGINX was to replicate the prod behaviour locally. Otherwise there would definitely be many ways to simply serve the API over HTTPS directly from the Django container.